api/src/services/graphql/index.ts

```diff
@@ -3146,100 +3144,6 @@ export class GraphQLService {
 			});
 		}
 
-		if ('directus_activity' in schema.create.collections) {
-			schemaComposer.Mutation.addFields({
-				create_comment: {
-					type: ReadCollectionTypes['directus_activity'] ?? GraphQLBoolean,
-					args: {
-						collection: new GraphQLNonNull(GraphQLString),
-						item: new GraphQLNonNull(GraphQLID),
-						comment: new GraphQLNonNull(GraphQLString),
-					},
-					resolve: async (_, args, __, info) => {
-						const service = new ActivityService({
-							accountability: this.accountability,
-							schema: this.schema,
-						});
-
-						const primaryKey = await service.createOne({
-							...args,
-							action: Action.COMMENT,
-							user: this.accountability?.user,
-							ip: this.accountability?.ip,
-							user_agent: this.accountability?.userAgent,
-							origin: this.accountability?.origin,
-						});
-
-						if ('directus_activity' in ReadCollectionTypes) {
-							const selections = this.replaceFragmentsInSelections(
-								info.fieldNodes[0]?.selectionSet?.selections,
-								info.fragments,
-							);
-
-							const query = this.getQuery(args, selections || [], info.variableValues);
-
-							return await service.readOne(primaryKey, query);
-						}
-
-						return true;
-					},
-				},
-			});
-		}
```

Comment:
I think these should be updated to use the CommentsService rather than be deleted. We still want people to be able to create and update comments over graphql

Reply:
The new `directus_comments` is usable via `/graphql/system` with a query like the following:

```gql
{
  comments (limit: 5) {
    id
    comment
    date_created
    user_created{
      first_name
    }
  }
}
```